web/components/dialogs/FcDialogRequestFilters.vue

```diff
@@ -78,30 +73,6 @@
             :value="assignee"></v-checkbox>
         </fieldset>
 
-        <FcRadioGroup
```

Comment:
same here - I've removed the `createdAt` and `lastEditedAt` filters, mostly because these would have been *hard* to set up with `study_request_items`.

the issue here is that I haven't found an easy way to build queries of the form "is any element in this array within this range?" - we'd have to do something like `unnest()`, but then we'd likely end up doing multiple queries and merging results from them, and that's even more complexity.